packages/blockchain-extension/extension/commands/testSmartContractCommand.ts

```diff
@@ -535,7 +535,7 @@ async function updateBuildFile(buildFile: string, state: any): Promise<void> {
             'fabric-gateway-java': {
                 groupId: 'org.hyperledger.fabric',
                 artifactId: 'fabric-gateway-java',
-                version: '1.4.2',
+                version: '2.1.0',
```

Comment:
It wasn't previously, but should `version: FABRIC_NETWORK_VERSION`?

Reply:
good point but i think we need separate variables for java and node